oh_staff_ui/models.py

```diff
@@ -17,3 +18,34 @@ class ItemType(models.Model):
 
     def __str__(self):
         return self.type
+
+
+class ProjectItem(models.Model):
+
+    ark = models.CharField(max_length=40, blank=False, null=False)
+    create_date = models.DateField()
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        related_name="+",
+    )
+    last_edit_date = models.DateField()
+    last_modified_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        related_name="+",
+    )
+    parent = models.ForeignKey("self", on_delete=models.SET_NULL, blank=True, null=True)
+    sequence = models.IntegerField(blank=True, null=True)
```

Comment:
Is it logical to have `sequence` be nullable?  If one item exists, it seems like it should have a `sequence` of `1`.  That also avoids possible sorting surprises: queries which order by sequence don't have to worry about NULL-first vs NULL-last behavior.

Reply:
Agreed, set to non-null and default of 1 if missing